python/fbprophet/forecaster.py

```diff
@@ -188,7 +188,7 @@ def validate_column_name(self, name, check_holidays=True,
         reserved_names = [
             'trend', 'additive_terms', 'daily', 'weekly', 'yearly',
             'holidays', 'zeros', 'extra_regressors_additive', 'yhat',
-            'extra_regressors_multiplicative', 'multiplicative_terms',
+            'extra_regressors_multiplicative', 'multiplicative_terms'
```

Comment:
We tend to leave these commas in so that if we add new rows it doesn't create a line diff.